### ISSUE TITLE
which one make sense?

### ISSUE THREAD

**Issue by KevinYuk** [raghakot/keras-resnet#37]
Hi,

I port the [resnet18 model](https://github.com/vgenty/resnet18/blob/master/ResNet-18-train_val.prototxt) from caffe to keras. Test is on CIFAR10. The convergence curve is below. The accuracy has also reached ~84%. But the curve is strange. Does it make sense?
![image](https://cloud.githubusercontent.com/assets/20220399/23977711/b821c908-0a2a-11e7-8f2d-9a54534a7c21.png)

And I have already recreate the keras-resnet here by CIFAR18:
![image](https://cloud.githubusercontent.com/assets/20220399/23977728/df193e60-0a2a-11e7-8b9f-4ed56aa551bc.png)

Thanks a lot.

**Comment by raghakot** [comment]
For the first one, it looks like it might be overfitting after epoch 30 ish...
I would try lowering learning rate.

The second one looks better imo. 
Did you drop the learning rate around 45th epoch?
